### PR TITLE
Added new 2019 Python Video

### PR DESCRIPTION
**Pull request by Akashd7**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -141,3 +141,7 @@ Inspired by [js-must-watch](https://github.com/bolshchikov/js-must-watch). Creat
     * [PyCon presentation](https://us.pycon.org/2015/schedule/presentation/362/)
     * Video: [youtube](https://www.youtube.com/watch?v=_AEJHKGk9ns) [00:25:20]/[pyvideo](http://pyvideo.org/video/3466/facts-and-myths-about-python-names-and-values)
     * [Slides and new article](http://nedbatchelder.com/text/names1.html)/[Original article](http://nedbatchelder.com/text/names.html)
+
+##2019
+
+https://courses.edx.org/courses/course-v1:MITx+6.00.1x+2T2019a/c
```
Comment (htlcnn) [inline README.md]: Not found

Comment (Akashd7) [reply README.md]: Hi . thanks for pointing out. have to remove the "C" at the end thats it.

Comment (Akashd7) [reply README.md]: Done.

**Comment by s16h** [comment]
I'm going to close this as I don't believe these are must-watch videos. Trying to keep this repo to a curated list of "universally awesome" Python videos. Please challenge me if you don't agree.

**Pull request closed without merging**